recipes/mailio/all/patches/adapt-cmakelists.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
I want to reduce patching to the minimum amount possible - this makes the recipe really nice and clean and generally improves the lib itself.

So I think I will create several PRs in the library repository, then I will ask the author to create a new version and add I will add conan recipe for that version.